compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -299,8 +299,15 @@ OMR::X86::CodeGenerator::initialize(TR::Compilation *comp)
    self()->setLastGlobalGPR(self()->machine()->getLastGlobalGPRRegisterNumber());
    self()->setLast8BitGlobalGPR(self()->machine()->getLast8BitGlobalGPRRegisterNumber());
    self()->setLastGlobalFPR(self()->machine()->getLastGlobalFPRRegisterNumber());
-   self()->setFirstGlobalVRF(self()->getFirstGlobalFPR());
-   self()->setLastGlobalVRF(self()->getLastGlobalFPR());
+   /*
+    * GRA does not work with vector registers on 32 bit due to a bug where xmm registers are not being assigned
+    * This will temporary disable GRA for vector registers on 32 bit until the problem has been resolved
```

Comment:
* Please use periods at the end of your sentences rather than just line breaks.
* Rather than say this fix is temporary, please reference the issue you created to track the resolution of the problem in the comment instead.  Part of the work in that issue should be to re-enable this code and remove the comment.